### PR TITLE
Update entities, Add project endpoint

### PR DESCRIPTION
**Pull request by ryonryon**

### CODE REVIEW

File: src/controllers/project.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { FastifyRequest, FastifyReply } from "fastify";
+import { Project } from "../entities";
+
+export default async function project(
```
Comment (kei95) [inline src/controllers/project.ts:4]: [Question] - Why is this function is named just `project`? Is this some sort of convention you follow? It's a bit counter-intuitive to scatter around requests with files for me. The pattern that I'm familiar with is to couple endpoints' function feature/entity based on entities. In this case, I'd put `projects` function in this file and name it like `getProjects` and `getProjectById` for this one. If that makes sense 🤔

Comment (kei95) [reply src/controllers/project.ts:4]: Did a little research, and came across the repo for Node.js best practices. The pattern I mentioned above is so called 3 layers pattern (architecture), and it's not recommended anymore. What they suggest is to put boundaries between components(features) to make it more concerned with its feature. Our project is so small that we won't really need to be worried or strictly follow this pattern, but I still think making files for each API is a bit too aggressive segregation.

The rule: https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/breakintcomponents.md

**Review by kei95 — APPROVED** [review]

**Comment by kei95** [comment]
Approved with some comments and questions about architecture. Once you check them, feel free to merge the PR or make changes if you would!

File: src/controllers/project.ts (hunk 1 above)
Comment (ryonryon) [reply src/controllers/project.ts:4]: lol, I am too used to GraphQL patterns. you are right. Let's decide which pattern would suit us. then we can update them.

Comment (kei95) [reply src/controllers/project.ts:4]: haha sure

**Pull request merged**